AutoRunCustomTool/AutoRunCustomTool.csproj

```diff
@@ -61,35 +61,16 @@
     <Reference Include="WindowsBase" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="Guids.cs" />
-    <Compile Include="GlobalSuppressions.cs" />
-    <Compile Include="AutoRunCustomToolPackage.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
```

Comment:
Shouldn't this file be moved to the shared project as well?

Reply:
Yes you're right. This has the benefit of updating the assembly details of the new VS2022 dll as well. I will move this.